src/utils/general_look_at_data.py

```diff
@@ -85,19 +82,13 @@ def plot_indirect_missing(missing_data):
     # Filter features where the percentage is greater than zero
     filtered_data = missing_data[missing_data.gt(0.0).any(axis=1)].drop(columns='feature_type')
 
-    # Create a list of features that have non-zero missing data
-    non_zero_features = list(filtered_data.columns[filtered_data.any()])
-
     # Create a bar plot of the missing data percentage for each feature
-    num_features = len(non_zero_features)
-    if num_features == 0:
-        print("No features found with missing data")
-        return
+    num_features = len(filtered_data.columns)
     num_cols = 2
     num_rows = math.ceil(num_features / num_cols)
     fig, axs = plt.subplots(num_rows, num_cols, figsize=(10, 5*num_rows))
 
-    for i, column_name in enumerate(non_zero_features):
+    for i, column_name in enumerate(filtered_data.columns):
```

Comment:
why did you update this code?

Reply:
Not me 😃

Reply:
Last change on general_look_at_data.py was before merge the Feature transformation pull request to main